cluster/images/controller-manager/Dockerfile

```diff
@@ -14,7 +14,7 @@
 ##                               BUILD ARGS                                   ##
 ################################################################################
 # This build arg allows the specification of a custom Golang image.
-ARG GOLANG_IMAGE=golang:1.13.14
+ARG GOLANG_IMAGE=golang:1.15.6
```

Comment:
Is this the GO version used in v1.19.7?

Reply:
@andrewsykim 
I took a look at https://github.com/kubernetes/kubernetes/blob/v1.19.7/go.mod#L9 and it's using `go 1.15` so I just search the latest Golang image version 1.15.6 and use it here, is that the right way to update it?

Reply:
yeah that seems good enough, just want to make sure we are on the same minor at least. Patch version is _probably_ not as important.